Library/Homebrew/cask/lib/hbc/dsl/appcast.rb

```diff
@@ -9,6 +11,24 @@ def initialize(uri, parameters = {})
         @checkpoint     = @parameters[:checkpoint]
       end
 
+      def calculate_checkpoint
+        result = SystemCommand.run("/usr/bin/curl", args: ["--compressed", "--location", "--user-agent", URL::FAKE_USER_AGENT, @uri], print_stderr: false)
+
+        checkpoint = if result.success?
+          processed_appcast_text = result.stdout.gsub(%r{<pubDate>[^<]*</pubDate>}, "")
+
+          # This step is necessary to replicate running `sed` from the command line
```

Comment:
I’d say these can be removed. If this exists to give the same result the non-internal command does, but the non-internal command will ceased to be used in favour of this internal one, then there’s nothing to replicate. This will be the canonical and only recommended way to do it.